examples/NativeExamples/main.cpp

```diff
@@ -6,10 +6,11 @@
 int main(void)
 {
     // Comment in/out the particular example you'd like to run.
-    kernel_and_user_trace_001::start();
+    //kernel_and_user_trace_001::start();
```

Comment:
let's leave example 001 as the non-commented-out method call here

Reply:
Done